src/test/java/org/openrewrite/hibernate/MigrateToHibernate61Test.java

```diff
@@ -155,5 +156,204 @@ public class TestApplication {
+
+                    import javax.persistence.Basic;
+                    import javax.persistence.Column;
+                    import javax.persistence.Entity;
+                    import javax.persistence.Id;
+                    import javax.persistence.Table;
+                    import java.io.Serializable;
+                    import java.util.Map;
+
+                    @Entity
+                    @Table(name = "schema_revisions")
+                    @TypeDefs({@TypeDef(name = "json", typeClass = JsonStringType.class)})
+                    public class SchemaRevisionEntity implements Serializable {
+                        @Id
+                        @Basic(optional = false)
+                        @Column(name = "name", nullable = false, updatable = false)
+                        private String name;
+
+                        @Type(type = "json")
+                        @Column(name = "schema_metadata", nullable = false)
+                        private SchemaRevisionMetadataEntity metadata;
+
+                        public static class SchemaRevisionMetadataEntity implements Serializable {
+                            private Map<String, String> userMetadata;
+                        }
+                    }
+                  """,
+                """
+                    import io.hypersistence.utils.hibernate.type.json.JsonType;
+                    import org.hibernate.annotations.Type;
+                      
+                    import jakarta.persistence.Basic;
+                    import jakarta.persistence.Column;
+                    import jakarta.persistence.Entity;
+                    import jakarta.persistence.Id;
+                    import jakarta.persistence.Table;
+                    import java.io.Serializable;
+                    import java.util.Map;
+                      
+                    @Entity
+                    @Table(name = "schema_revisions")
+                    public class SchemaRevisionEntity implements Serializable {
+                        @Id
+                        @Basic(optional = false)
+                        @Column(name = "name", nullable = false, updatable = false)
+                        private String name;
+                      
+                        @Type(value = JsonType.class)
+                        @Column(name = "schema_metadata", nullable = false)
+                        private SchemaRevisionMetadataEntity metadata;
+                      
+                        public static class SchemaRevisionMetadataEntity implements Serializable {
+                            private Map<String, String> userMetadata;
+                        }
+                    }
+                  """
+              )
+            )
+          )
+        );
+    }
+
+    @Test
+    void typeDefsAnnotationIsNotRemovedWhenHypersistenceUtilsAreNotBeingUsed() {
+        rewriteRun(
+          spec -> spec.typeValidationOptions(TypeValidation.none()),
```

Comment:
Same applies to this test.

Reply:
You might also need to pass in a minimal `SomeCustomClass`, as that's used in the `SchemaRevisionEntity`, but not defined in the sources that are passed into the test.